setup.py

```diff
@@ -13,9 +16,10 @@
     long_description=open("README.md", "r").read(),
     long_description_content_type="text/markdown",
     license="CC0",
-    version="4.2.1",
+    version="4.2.2",
```

Comment:
We don't need to do a new release for this change, so let's leave the version alone.

Reply:
Done